docs/pages/access-controls/access-graph/self-hosted.mdx

```diff
@@ -1,20 +1,14 @@
 ---
-title: Run Teleport Access Graph on Self-Hosted Clusters
-description: Describes how to deploy Teleport Access Graph on self-hosted clusters.
+title: Run Teleport Policy on Self-Hosted Clusters
```

Comment:
Feels like this title should be more similar to the title in self-hosted-helm.mdx.